tools/schema/cardBaseSchema.json

```diff
@@ -54,18 +54,15 @@
             "maxLength": 20,
             "pattern": "^[a-z]+_[0-9a-z]+$"
           },
+          "direction": {
+            "description": "Link direction; either towards this card, or outwards from this card.",
+            "type": "string",
+            "enum": ["outbound", "inbound"]
+          },
           "linkType": {
-            "description": "The link type is named according to the from-to direction. In user interfaces, the naming according to the link type is shown in the 'from' card.",
+            "description": "The link type is named according to the 'direction'. In user interfaces, the naming according to the link type is shown in the 'from' card.",
```

Comment:
The description should be "The name of the link type. For example \"base/linkTypes/causes\"." because we are now using name link types instead of a hard-code enum.